src/main/java/com/rocketden/main/dto/problem/CreateTestCaseRequest.java

```diff
@@ -1,4 +1,12 @@
 package com.rocketden.main.dto.problem;
 
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
 public class CreateTestCaseRequest {
+    private String input;
+    private String output;
```

Comment:
Not sure - the difficulty with custom objects like trees, maps, arrays, etc. is that those things don't really convert into database tables/rows very well, and also idk how to allow users to create Java objects on the frontend if they want to write custom test cases (like LeetCode does). However I'm not too familiar so I def think it's possible depending on what we go with for the compiling process. 

As for images, I think it would be easiest if those just go in the problem description section (which maybe could be formatted as markdown?) since the majority of test cases will likely be hidden or auto generated or something, but again this totally depends on how we go about it I think

Also, remember [this](https://github.com/DrDab/LaunchTestRun) haha? Might be a useful reference (although this uses scanners instead of method args)

Reply:
That first point is fair -- we can keep strings for input and output, at least in this PR.

Yeah I had meant to bring up LaunchTestRun -- could we reach out to him and ask how that works and how to set ours up potentially?

Reply:
I think that could be worth looking into